crates/yttrium/src/uniffi_compat/sign/storage.rs

```diff
@@ -42,6 +43,8 @@ pub trait StorageFfi: Send + Sync {
         topic: String,
         sym_key: Vec<u8>,
     ) -> Result<(), StorageError>;
+    fn get_verify_public_key(&self) -> Result<Option<String>, StorageError>;
+    fn set_verify_public_key(&self, jwk: String) -> Result<(), StorageError>;
```

Comment:
So native clients should store the jwt?

Reply:
It's JWK not JWT and yes

I see you guys are storing just the bytes of the public key, but string JWK would be slightly cooler if we could do that.